keyboards/jj4x4/keymaps/via/keymap.c

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2020
```

Comment:
Any particular reason you didn't copy the default keymap for this?

Reply:
The default keymap has media keys on it, which don't work with V-USB boards just yet. Instead I used the default ortho_4x4 layout as reference.